### DISCUSSION TITLE
Some of My JS Tricks to Enhance the HTML Output of Markdown | /en/2018/11/md-js-tricks/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1485] [Comments]
Just because Markdown is so simple (well, &ldquo;simple&rdquo;), there will definitely be useful features missing. I want to share some of my relatively simple JS tricks to fill the gaps. Please note &hellip;

https://yihui.org/en/2018/11/md-js-tricks/

**Comment by atlas555** [comment]
interesting tools

**Comment by hongtaoh** [comment]
I don't think there were bicycles in Tang Dynasty, lol.
